frontend/src/components/AutoComplete/AutoComplete.jsx

```diff
@@ -1,39 +1,91 @@
-import React from 'react';
+import React, { useState, useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import Autocomplete from '@material-ui/lab/Autocomplete';
 import TextField from '@material-ui/core/TextField';
+import { debounce } from 'lodash/fp';
```

Comment:
A suggestion in order to import only what we need

```suggestion
import debounce from 'lodash/fp/debounce';
```